implementations/rust/ockam/ockam_identity/src/lib.rs

```diff
@@ -18,6 +18,12 @@ extern crate core;
 #[macro_use]
 extern crate alloc;
 
+#[cfg(test)]
+extern crate quickcheck;
+#[cfg(test)]
+#[macro_use(quickcheck)]
```

Comment:
Is `#[macro_use]` even a thing now? I thought you don't need it anymore

Reply:
Actually this seems to be necessary: https://github.com/build-trust/ockam/actions/runs/4142577738/jobs/7163412333